offer-manager-app/src/main/groovy/life/qbic/portal/offermanager/dataresources/persons/CustomerDbConnector.groovy

```diff
@@ -288,6 +300,53 @@ class CustomerDbConnector implements CreateCustomerDataSource, UpdateCustomerDat
     }
     return affiliationIds[0]
   }
+  
+  @Override
+  void updateCustomerAffiliations(String customerIdString, List<Affiliation> updatedAffiliations) {
```

Comment:
I would expect to just get one affiliation and not a list. When we create a customer we obtain only a single affiliation and when updating we can add single affiliations